### PR TITLE
LPD-20229 Add New Solutions Base Flow

### PR DESCRIPTION
**Pull request by joanagrego**

### CODE REVIEW

**Comment by joanagrego** [comment]
ci:test:sf

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/AdministratorDashboard/components/LicenseAgreement/LicenseAgreement.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+				<small>
+					PLEASE READ THE FOLLOWING LIFERAY PUBLISHER PROGRAM LICENSE
+					AGREEMENT TERMS AND CONDITIONS CAREFULLY BEFORE DONLOADING
+					OR USING THE LIFERAY SOFTWARE OR LIFERAY SERVICES. THESE
+					TERMS AND CONDITIONS CONSTITUTE A LEGAL AGREEMENT BETWEEN
+					YOU AND LIFERAY.
+				</small>
+
+				<br />
+				<br />
```
Comment (welitonpe) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/AdministratorDashboard/components/LicenseAgreement/LicenseAgreement.tsx:31]: Could you use a bootstrap class to add this margin?

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
Resent --> [PR](https://github.com/is-solutions-delivery/liferay-portal/pull/5618)

**Pull request closed without merging**